src/test/shell/testenv.sh

```diff
@@ -588,9 +588,12 @@ function add_zlib() {
 }
 
 function add_protobuf() {
-  version=$(get_version_from_default_lock_file "protobuf")
+  protobuf_version=$(get_version_from_default_lock_file "protobuf")
+  abseil_version=$(get_version_from_default_lock_file "abseil-cpp")
```

Comment:
The version of abseil version from the default lock file is from MODULE.tools, which is basically decided by protobuf, I see you have upgraded abseil-cpp in the root MODULE.bazel, which is a newer than the one protobuf introduced.

Reply:
I'll update the version in MODULE.tools. If it affects our tests, it would also affect users 🙂